code/graphics/opengl/gropenglshader.cpp

```diff
@@ -327,6 +337,8 @@ void opengl_delete_shader(int sdr_handle)
 {
 	Assert(sdr_handle >= 0);
 	Assert(sdr_handle < (int)GL_shader.size());
+	opengl_shader_t &victim = GL_shader[sdr_handle];
+	GL_shader_map.erase(std::make_pair(victim.shader, victim.flags));
```

Comment:
I think you should use the `shader_descriptor_t` here:
```suggestion
	GL_shader_map.erase(shader_descriptor_t(victim.shader, victim.flags));
```